.github/workflows/main.yml

```diff
@@ -0,0 +1,34 @@
+name: CI/CD Pipeline
+
+on:
+  push:
+    branches:
+      - '**'
+  pull_request:
+    branches:
+      - main
+
```

Comment:
Please also add `develop` in the list of branches, so that it is both `main` and `develop`